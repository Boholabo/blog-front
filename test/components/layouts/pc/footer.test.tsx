import React from 'react';
import { render } from '../../../testUtils';
import Footer from '../../../../components/layouts/pc/footer';

describe('layouts/pc/footer component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
