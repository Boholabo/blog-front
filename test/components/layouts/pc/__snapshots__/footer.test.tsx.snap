// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layouts/pc/footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
    id="footer"
  >
    <div
      class="container"
    >
      <span>
        Powered By
      </span>
      <a
        class="item"
        href="blog"
        rel="noreferrer nofollow noopener"
        target="_blank"
      >
        blog
      </a>
      <span>
        <a
          class="item"
          href="https://boholabo.com"
          rel="noreferrer nofollow noopener"
          target="_blank"
        >
          Boholabo
        </a>
        <a
          href="/about"
        >
          Me
        </a>
      </span>
    </div>
  </footer>
</DocumentFragment>
`;
